README.md

```diff
@@ -14,44 +14,79 @@ Requires:
 1. Checkout the code and install the dependencies:
 
   ```
-  > git clone git@github.com:umd-lib/archelon.git
-  > cd archelon
-  > bundle install
+  git clone git@github.com:umd-lib/archelon.git
+  cd archelon
+  bundle install
   ```
 
 2. Set up the database:
 
   ```
-  > rake db:migrate
+  rake db:migrate
   ```
 
   **Note:** Sample "Download URL" data can be added by running `rake db:reset_with_sample_data`
 
-3. Create a `.env` file from the `env_example` file and set the solr url to point to a working solr url.
+3. Create a `.env` file from the `env_example` file and fill in appropriate values for the environment variables.
 
 4. Add your directory ID to whitelist
 
   ```
-  > rake 'db:add_admin_cas_user[your_directory_id, Your Name]'
+  rake 'db:add_admin_cas_user[your_directory_id, Your Name]'
   ```
 
 5. Run the web application:
 
   ```
-  > rails server
+  rails server
   ```
 
 If you are going to run Archelon against a Solr or Fedora server that uses self-signed SSL certificates for HTTPS, see the section [SSL setup](#ssl-setup).
 
 See [archelon-vagrant] for running Archelon application in a Vagrant environment.
 
+## Docker
+
+Archelon comes with a [Dockerfile](Dockerfile) that can be used to build a docker image:
+
+```
+docker build -t archelon .
+```
+
+To run an instance of this image against the dev fcrepo, Solr, and IIIF servers, and populate the database with seed data:
+
+```
+id=$(docker run -d --rm -p 3000:3000 \
```

Comment:
Alternatively, If we include a `--name archelon` argument here, we can use that instead of `id` for other docker commands (Eg. `docker kill archelon`). But, we need to ensure that we use different names if we want to run multiple instances. So, I am ok with leaving it this way, if you prefer.

Reply:
I'd like to leave it this way (with the `$id` variable).